readme.txt

```diff
@@ -4,10 +4,10 @@ Contributors:      wordpressdotorg
 Requires at least: 6.4
 Tested up to:      6.5
 Requires PHP:      7.0
-Stable tag:        2.9.0
+Stable tag:        3.0.0
 License:           GPLv2 or later
 License URI:       https://www.gnu.org/licenses/gpl-2.0.html
-Tags:              performance, images, javascript, site health, measurement, object caching
+Tags:              performance, site health, measurement, object caching
```

Comment:
I ran the `readme.txt` through the WordPress.org readme validator and it warned about there being more than 5 tags. Since the functionality for `images` and `javascript` are now in standalone plugins, I'm removing them here.

Reply:
@westonruter Let's also remove `object caching` based on that rationale. Those 5 tags were based on our initial "focus areas". But the only two areas that the PL plugin now covers on its own is Site Health and measurement.

Reply:
Done in 9461d57b

Reply:
Added 2 more tags based on suggestions from Gemini to bring the total back up to: 59a75894